reference/constraints/Uuid.rst

```diff
@@ -48,6 +40,14 @@ Basic Usage
              protected $identifier;
```

Comment:
off topic but shouldn't we change properties to `private`?

Reply:
Probably yes ... but, as you said, off topic :)